src/lib/netlist/build/create_devinc.py

```diff
@@ -72,6 +79,11 @@ def process_file(srcfile):
                         m = re.match(r"(static)*TRUTHTABLE_START\((\w+),(\w+),(\w+),([a-zA-Z0-9_+@,]*)", ls)
                         if m != None:
                             process_entry(srcfile, m.group(2), m.group(5))
+                        else:
+                            #m = re.match(r"EXTERNAL_SOURCE\((\w+)\)", ls)
+                            m = re.match(r"NETLIST_START\((\w+)\)", ls)
+                            if m != None:
```

Comment:
You probably want `m is not None` here – equality comparison with `None` can have weird semantics, you generally want to check whether the thing is or isn’t the special `None` object.

Reply:
Applied. Thanks.